movies.json

```diff
@@ -0,0 +1,802 @@
+[
```

Comment:
This file shouldn't have been committed, make sure you are always keeping github clean.